libfuse/fs.go

```diff
@@ -257,3 +274,26 @@ func (r *Root) ReadDirAll(ctx context.Context) (res []fuse.Dirent, err error) {
 	}
 	return res, nil
 }
+
+func (r *Root) log() logger.Logger {
+	return r.private.fs.log
+}
+
+var loadVolIconOnce sync.Once
```

Comment:
no need for this to be a global. It can live inside `Root`. Or better yet, this can be made part of `newExternalFile` (see comment above)